app/routes/claim/view-claim.js

```diff
@@ -87,12 +88,19 @@ module.exports = function (router) {
   router.get('/claim/:claimId/download', function (req, res) {
     authorisation.isCaseworker(req)
 
-    var path = req.query.path
-    if (path) {
-      res.download(path)
-    } else {
-      throw new Error('No path to file provided')
+    var claimDocumentId = req.query['claim-document-id']
+    if (!claimDocumentId) {
```

Comment:
No validation no. Yes ClaimDocumentID is just an integer sequence. I think it would be a good idea to alter it to be a generated value similar to reference rather than a simple sequence to prevent guessing of ids. I considered that out of scope for this story though.